src/main/core/manager.c

```diff
@@ -212,31 +213,31 @@ Manager* manager_new(Controller* controller, const ConfigOptions* config, Simula
         trace("raw manager cpu frequency unavailable, using 2,500,000 KHz");
     }
 
-    manager->preloadShimPath = _manager_scanRPathForLib(PRELOAD_SHIM_LIB_STR);
-    if (manager->preloadShimPath != NULL) {
-        info("found required preload library %s at path %s", PRELOAD_SHIM_LIB_STR,
-             manager->preloadShimPath);
+    manager->preloadDummyPath = _manager_scanRPathForLib(PRELOAD_DUMMY_LIB_STR);
+    if (manager->preloadDummyPath != NULL) {
+        info("found preload library %s at path %s", PRELOAD_DUMMY_LIB_STR,
+             manager->preloadDummyPath);
     } else {
-        // The shim is required, so panic if we can't find it.
-        utility_panic("could not find required preload library %s in rpath", PRELOAD_SHIM_LIB_STR);
+        // The dummy preload is only used if we have no other preloads configured.
+        warning("could not find preload library %s in rpath", PRELOAD_DUMMY_LIB_STR);
```

Comment:
Why did this change to a warning? It seems like this should be required in case the libc or openssl preloads aren't found or enabled.

Reply:
Sorry, that was copied and didn't get updated properly on my rebase. In any case, the path checking logic has been updated now to always panic if required libs are not found.